### DISCUSSION TITLE
Factory defaults sound like garbage?

### DISCUSSION THREAD

**Discussion by AugsEU** [bangcorrupt/hacktribe#115] [Q&A] [answered]
After updating my electribe the defaults now sounds really weird and out of tune. Anyone had this before, is it normal?

**Comment by maks** [comment, marked as answer]
Do you mean the built in patterns? If so I think that's to be expected as the sample slots no longer have the factory samples in them.

**Comment by AugsEU** [comment]
Oh OK. Is there any way to get the factory drum sounds again?

**Reply by bangcorrupt** [reply]
[FAQ - Where are the factory samples?](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#where-are-the-factory-samples)

#61

**Reply by AugsEU** [reply]
I have the synth version which came with some drum sounds I think.

**Reply by bangcorrupt** [reply]
#77

**Reply by AugsEU** [reply]
Oh OK. Thanks.
